package.json

```diff
@@ -36,6 +36,7 @@
     "eslint": "^2.10.2",
     "istanbul": "^0.4.3",
     "mocha": "^2.5.3",
+    "proxyquire": "^1.7.9",
```

Comment:
You can add `var FormController = require('hmpo-form-controller');` then in beforeEach add `sinon.stub(FormController.prototype, 'getForkTarget');` and in afterEach add `FormController.prototype.getForkTarget.restore();`.